File: java/doit_java/src/chap11/section4/classex/Person.java
// Person 클래스 생성하기
package chap11.section4.classex;

public class Person {
    private String name;
    private int age;

    // 디폴트 생성자
    public Person() {}

    // 이름만 입력받는 생성자
    public Person(String name) {
        this.name = name;
    }

    // 이름과 나이를 입력받는 생성자
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }
}

File: python/p_oop2/chap05/excercise/person.py
class Person:
    # 멤버변수 선언
    age = None
    name = None
    is_married = None
    number_of_children = None


def main():
    person = Person()
    person.age = 40
    person.name = "James"
    person.is_married = True
    person.number_of_children = 3

    print("나이:", person.age)
    print("이름:", person.name)
    print("결혼 여부:", person.is_married)
    print("자녀 수:", person.number_of_children)


if __name__ == '__main__':
    main()
